src/lib/process/index.ts

```diff
@@ -55,9 +57,29 @@ export const createProcess = async (
   const polkadot: Polkadot.Polkadot = await createNodeApi(options)
   const expectedVersion: number = (await getVersion(polkadot, processId)) + 1
 
-  if (version !== expectedVersion) {
+  // TODO simplify
+  if (version < 1 || version > expectedVersion) {
     throw new VersionError(`Version: ${version} must be incremented to: ${expectedVersion}`)
   }
+  
+  if (version !== expectedVersion) {
```

Comment:
my understanding of the ticket is that we only want to run this check if the version number that the user is attempting to create exactly matches the version number on chain